### PR TITLE
ability to change scaling

### PR DESCRIPTION
**Pull request by ssharpe42**
Add parameter to change scaling or use raw data

### CODE REVIEW

File: synthetic_data/synthetic_data.py [hunk 1]
```diff
@@ -289,7 +295,9 @@ def make_tabular_data(
     #
 
     if noise_level_x > 0.0:
-        x_noise = generate_x_noise(x_final[:, :n_informative], noise_level_x, seed=seed)
```
Comment (cbbruss) [inline synthetic_data/synthetic_data.py:292]: appreciate the linting

File: synthetic_data/synthetic_data.py [hunk 2]
```diff
@@ -269,8 +274,9 @@ def make_tabular_data(
     #        x_final = x_cont
 
     # rescale to -1 to 1
-    scaler = MinMaxScaler(feature_range=[-1, 1])
-    x_final = scaler.fit_transform(x_final)
+    if feature_range is not None:
```
Comment (cbbruss) [inline synthetic_data/synthetic_data.py:277]: Previously all data was rescaled. Now all data is only rescaled if the feature_range parameter is set. Do we always want every feature to be rescaled with the same range? Not an issue with this implementation but perhaps a ticket worth discussing.

File: synthetic_data/synthetic_data.py [hunk 3]
```diff
@@ -206,7 +219,7 @@ def make_tabular_data(
         p_thresh - probability threshold for assigning class labels
         noise_level_x (float) - level of white noise (jitter) added to x
         noise_level_y (float) - level of white noise added to y (think flip_y)
-        feature_range (list) - list of lower and upper range for feature scaling (e.g., [0,1]).
+        scaler (sklearn scaler) - sklearn style scaler. Defaults to MinMaxScaler(feature_range = (0,1)).
```
Comment (cbbruss) [inline synthetic_data/synthetic_data.py:222]: Looks like the default is (-1, 1)

Comment (tazitoo) [reply synthetic_data/synthetic_data.py:222]: Yup - for clean synthetic data with no outliers, the initial design consideration was to have only 1 scaler, so the range [-1,1] was the most straightforward.  And should probably remain the default.

Comment (cbbruss) [reply synthetic_data/synthetic_data.py:222]: Right, need to change the value in the docstring which states (0, 1)

**Comment by tazitoo** [comment]
Can you 
1. Remove the dependency on sympy in your version of  tests/test_synthetic_data.py 
2. Make a new environment from requirements.txt (no sympy needed now) and check that the tests complete

**Review by tazitoo — APPROVED** [review]
LGTM!

**Pull request merged**